R/config.r

```diff
@@ -129,14 +129,8 @@ curl_docs <- function(x) {
   BROWSE(url)
 }
 
-default_ua <- function() {
-  versions <- c(
-    libcurl = curl::curl_version()$version,
-    `r-curl` = as.character(packageVersion("curl")),
-    httr = as.character(packageVersion("httr"))
-  )
-  paste0(names(versions), "/", versions, collapse = " ")
-}
+# This is defined in .onLoad()
+default_ua <- NULL
```

Comment:
Could you please keep all the code here? I'd prefer you did something like 

```R
cache <- new.env(parent = emptyenv())
cache$default_ua <- NULL
```

and then returned early in `default_ua()` if that value was set